# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/base/nsIncrementalStreamLoader.cpp
# Commit: 3b150d676b31
# Full Hash: 3b150d676b3122bffa703813876d1fbc35e10d17
# Author: Randell Jesup <rjesup@mozilla.com>
# Date: 2024-03-19 16:41:28
# Regressor Bug: 1356686
# File Overlap Count: 1
# Description:
#   Bug 1356686 - Do decompression off main thread in content r=necko-reviewers,valentin,extension-reviewers,robwu
#   
#   Content processes will now always retarget delivery of OnDataAvailable for Http
#   channels off the main thread. Consumers that were previously redirecting
#   off-main thread are not affected and their retargeting will stick, but any
# ==============================================================================

diff -r 54754a50d992 -r 3b150d676b31 netwerk/base/nsIncrementalStreamLoader.cpp
--- a/netwerk/base/nsIncrementalStreamLoader.cpp	Mon Mar 18 20:04:00 2024 +0000
+++ b/netwerk/base/nsIncrementalStreamLoader.cpp	Mon Mar 18 20:04:01 2024 +0000
@@ -182,7 +182,9 @@
 void nsIncrementalStreamLoader::ReleaseData() { mData.clearAndFree(); }
 
 NS_IMETHODIMP
-nsIncrementalStreamLoader::CheckListenerChain() { return NS_OK; }
+nsIncrementalStreamLoader::CheckListenerChain() {
+  return NS_ERROR_NO_INTERFACE;
+}
 
 NS_IMETHODIMP
 nsIncrementalStreamLoader::OnDataFinished(nsresult aStatus) { return NS_OK; }